.pre-commit-config.yaml

```
@@ -16,6 +16,12 @@ repos:
       - id: mixed-line-ending
       - id: end-of-file-fixer
       - id: trailing-whitespace
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v3.3.1
+    hooks:
+      - id: pyupgrade
+        args: ["--py37-plus", "--keep-percent-format"]
+        exclude: ^(constructor/nsis/.*py)
```

```suggestion
        exclude: ^(constructor/nsis/*.py)
```

Did you mean to type this? :D

---

yes... this also confused me... it is not a glob pattern (then *.sh would be good), but a regex (where "." matches a single character 0 or any times "*").

---

Ah, wow, hahaha ok ok then this is ok